src/pages/learn/index.tsx

```diff
@@ -0,0 +1,943 @@
+                description="Learn about different ways to earn rewards by staking ETH."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.rhino)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/staking/">Start staking</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Run a node"
+                description="Run a node to play a crucial part in the Ethereum network."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.ethereumInside)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/run-a-node/">Run a node</ButtonLink>
+                </>
+              </Card>
+            </CardGrid>
+          </Section>
+
+          <Section>
+            <h2 id={tocItems[4].id}>{tocItems[4].title}</h2>
+            <p>
+              For users most interested in the technical part of the Ethereum
+              network.
+            </p>
+            <CardGrid>
+              <Card
+                title="Energy Consumption"
+                description="How much energy does it cost to run Ethereum network?"
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.hackathon)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/energy-consumption/">
+                    Is Ethereum green?
+                  </ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Ethereum upgrades"
+                description="How are we going to make Ethereum more scalable, secure, and sustainable?"
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.merge)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/upgrades/">Read more</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Ethereum Whitepaper"
+                description="The original Ethereum proposal written by Vitalik Buterin in 2014."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.financeTransparent)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/whitepaper/">Read whitepaper</ButtonLink>
+                </>
+              </Card>
+            </CardGrid>
+
+            <AdditionalReadingHeader>
+              More about the Ethereum protocol
```

Comment:
The other headers use the 'Additional reading on...' pattern. We should be consistent. Do we have a preference?

My slight preference is the 'More on...' style as it has a more coherent tone with the writing style/audience we're trying to reach